packages/test/test-utils/package.json

```diff
@@ -115,7 +115,6 @@
 	},
 	"dependencies": {
 		"@fluid-internal/test-driver-definitions": "workspace:~",
-		"@fluidframework/aqueduct": "workspace:~",
```

Comment:
This is the primary goal of this PR.

Reply:
is there a practical way to make our layering enforcement prevent this from being restored (directly or indirectly)?

Reply:
I'll double check this. I believe the current enforcement is by layer, rather than by package, so I don't _think_ we can do this. But I'll verify and enable if possible.

Reply:
No, it doesn't look like this is currently possible. Specifying allowed deps for the "Test-Utils" layer results in the following error:
```shell
 Error: ERROR: dev layers should not specify allowed deps since verification is skipped for dev layers (Test-Utils)
```

Adding support for dependency enforcement of "dev" layers seems valuable to me, so I am curious if this would be easy for us to add support. @tylerbutler maybe you can speak to this?

In the short term, getting a dependency from `aqueduct` on `tree` will have the side effect of preventing this dependency from returning via our dep cycle checks.

Reply:
I find the layer check code pretty hard to understand myself, so I would say changes are hard, but someone with a more algorithmic brain might feel differently. Code is here: https://github.com/microsoft/FluidFramework/blob/main/build-tools/packages/build-cli/src/library/layerGraph.ts

I'm sure we could replicate its functionality with some other tool, but I haven't ever investigated that.